Simplified.xcodeproj/project.pbxproj

```diff
@@ -2522,6 +2567,14 @@
 			path = AgeCheck;
 			sourceTree = "<group>";
 		};
+		177C25E328B6EDF300A786F1 /* Bookmark */ = {
```

Comment:
I can't tell where this group is exactly, but we already have a Bookmarks group under the "Reader2" folder

Reply:
This is under `Audiobooks`, I created it so the business logic and any future classes and stay under `Audiobooks/Bookmark/` instead of `Reader2/Bookmark`.